### ISSUE TITLE
Overview example for switch ranges now throwing compilation warnings...easy update on Overview

### ISSUE THREAD

**Issue by mahSource** [odin-lang/odin-lang.org#144]
Under this header <https://odin-lang.org/docs/overview/#switch-statement>
A switch statement can also use ranges like a range-based loop:
```odin
switch c {
case 'A'..'Z', 'a'..'z', '0'..'9':
	fmt.println("c is alphanumeric")
}
```
Compiler warning: **Syntax Warning: '..' for ranges has now been deprecated, prefer '..='**

Suggested update resulting in no warnings:
```odin
switch c {
	case 'A' ..= 'Z', 'a' ..= 'z', '0' ..= '9':
		fmt.println("c is alphanumeric")
	}
```

**Comment by Kelimion** [comment]
Closed via 1a2cb94b6c4050b2fcc7955f770f09af425ea921.

**Comment by mahSource** [comment]
Thank you!
